contracts/BootstrapRewards.sol

```diff
@@ -86,18 +86,19 @@ contract BootstrapRewards is IBootstrapRewards, ContractRegistryAccessor {
 
             uint256 duration = now.sub(lastPayedAt);
             uint256 amountPerGeneralValidator = Math.min(generalCommitteeAnnualBootstrap.mul(duration).div(365 days), pool.div(committee.length));
+            pool = pool.sub(amountPerGeneralValidator * committee.length);
```

Comment:
It's just 2, not times committee length. Also, it will be removed entirely in a future PR because we will not check the pool at this point but only on withdrawal.